cangjie5.dict.yaml

```diff
@@ -27368,7 +27369,6 @@ encoder:
 䩁	lyhuu
 䘻	lyib
 䘪	lyiu
-䫍	lmsyc
```

Comment:
`䫍	lmsyc` should not be placed here according to the Cangjie code order, it is moved from line 27371 to line 27300. No observable changes because `lmsyc` is unique.